README.md

```diff
@@ -1,2 +1,63 @@
+<!-- omit in toc -->
 # memory-qt
-Memory game in Qt
+
+- [Goal Of The Repository](#goal-of-the-repository)
+- [Description Of The Application](#description-of-the-application)
+- [Download Compiled Binaries](#download-compiled-binaries)
+- [How To Build](#how-to-build)
+- [Use Cases](#use-cases)
+- [Requirements](#requirements)
+- [Ideas for Improvement](#ideas-for-improvement)
+
+
+# Goal Of The Repository
+
+This repository describes the implementation of an open-source Memory game in Qt, applying principles of software architecture such as requirements gathering and object-oriented analysis and design.
+
+This project uses the [Qt Widgets? / Qt Quick? TBD] approach.
```

Comment:
After last meeting I'd say we can probably commit to Qt Quick?
```suggestion
This project uses the Qt Quick approach.
```

Reply:
Yes, I agree.

Reply:
> Yes, I agree. If @DanieleAsnicar also agrees, we can commit the suggestion and close the conversation.